templates/cisco_nxos_show_license_usage.textfsm

```diff
@@ -0,0 +1,15 @@
+Value FEATURE (\S+)
+Value INSTALLED (Yes|No)
+Value LICENSE_COUNT (\d+|-)
+Value STATUS (Unused|In\s+use)
+Value EXPIRY_DATE (\S+|\s*?)
+Value COMMENTS (.+)
+
+Start
+  ^\s+Count\s*$$ -> Begin
```

Comment:
```suggestion
Start
  ^Feature\s+Ins\s+Lic\s+Status\s+Expiry\s+Date\s+Comments - > Begin
  ^\s*$$
  ^. -> Error

Begin
  ^\s+Count

Reply:
Done

Reply:
I still see the same thing is was prior.

Reply:
The code has been updated like you suggested though, the update is visible in my forked repo. Is there anything else I need to do ?

Reply:
You need to commit and push it up to your branch on Github for it to update here.

Reply:
Woops, looks like I forgot to push on the branch indeed. Just fixed that.